sdk/customproviders/azure-mgmt-customproviders/azure/mgmt/customproviders/_configuration.py

```diff
@@ -0,0 +1,49 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+from msrestazure import AzureConfiguration
+
+from .version import VERSION
+
+
+class customprovidersClientConfiguration(AzureConfiguration):
```

Comment:
i wonder why it's lowercase

Reply:
Let me check

Reply:
https://github.com/Azure/azure-rest-api-specs/blob/14543d35a8474edcc8d0d9374fedef1047609107/specification/customproviders/resource-manager/Microsoft.CustomProviders/preview/2018-09-01-preview/customproviders.json#L8  because they have this

Reply:
I think it's fine for now

Reply:
we should fix it, otherwise we will introduce breaking change in the future